microsoftgraph/microsoft-graph-explorer-v2
# adds SdkVersion as default header

## Overview

Adds a default header SdkVersion to all requests.

Fixes #222 

## Testing Instructions

* Hit run query
* Check the network tab, and look at the request headers
* Note the addition of the GE-Version header
